apps/wallet/src/ui/app/shared/faucet/request-button/index.tsx

```diff
@@ -20,19 +20,27 @@ type FaucetRequestButtonProps = {
 function FaucetRequestButton({ mode = 'primary' }: FaucetRequestButtonProps) {
     const network = useAppSelector(({ app }) => app.apiEnv);
     const networkName = API_ENV_TO_INFO[network].name;
+    const showFaucetRequestButton = API_ENV.customRPC !== network;
     const dispatch = useAppDispatch();
     const loading = useAppSelector(({ faucet }) => faucet.loading);
     return (
-        <Button
-            mode={mode}
-            onClick={() => {
-                dispatch(requestGas());
-            }}
-            disabled={loading}
-        >
-            <Icon icon={SuiIcons.Download} className={cl(st.icon, st[mode])} />
-            Request {networkName} SUI Tokens
-        </Button>
+        <>
```

Comment:
Fragment is not necessary right?

Reply:
I thought that react does not allow the return of empty/null component.

Reply:
null should be fine

Reply:
Nope. does not work